[STCC-209] condition blocks were broken. Should be fixed now.

gregbauer opened the pull request:
Python Lists collided with Java Arrays; the inner brick lists of the Java Condition bricks (Repeat, while, if else etc.) where overwritten by Python Lists whereas they should have been Java Arrays.

sandra0521 commented:
I converted a view projects and they could be loaded properly in Pocket Code. Maybe we should merge this soon - would make testing a lot easier

AntiDog commented:
> I converted a view projects and they could be loaded properly in Pocket Code. Maybe we should merge this soon - would make testing a lot easier

did you also try nested loops?

gregbauer commented:
Updated the commit. Found out that some condition blocks with empty "body" (no inner bricks) caused the converter to throw an exception. Also removed an unnecessary line.
